nimbus-jose-jwt_aws-kms-extension/src/main/java/com/nimbusds/jose/aws/kms/crypto/utils/JWEDecrypterUtil.java

```diff
@@ -0,0 +1,73 @@
+package com.nimbusds.jose.aws.kms.crypto.utils;
+
+import com.amazonaws.services.kms.AWSKMS;
+import com.amazonaws.services.kms.model.DecryptRequest;
+import com.amazonaws.services.kms.model.DecryptResult;
+import com.amazonaws.services.kms.model.DependencyTimeoutException;
+import com.amazonaws.services.kms.model.DisabledException;
+import com.amazonaws.services.kms.model.InvalidGrantTokenException;
+import com.amazonaws.services.kms.model.InvalidKeyUsageException;
+import com.amazonaws.services.kms.model.KMSInternalException;
+import com.amazonaws.services.kms.model.KMSInvalidStateException;
+import com.amazonaws.services.kms.model.KeyUnavailableException;
+import com.amazonaws.services.kms.model.NotFoundException;
+import com.nimbusds.jose.JOSEException;
+import com.nimbusds.jose.JWEAlgorithm;
+import com.nimbusds.jose.JWEHeader;
+import com.nimbusds.jose.RemoteKeySourceException;
+import com.nimbusds.jose.aws.kms.exceptions.TemporaryJOSEException;
+import com.nimbusds.jose.crypto.impl.ContentCryptoProvider;
+import com.nimbusds.jose.util.Base64URL;
+import com.nimbusds.jose.jca.JWEJCAContext;
+import java.nio.ByteBuffer;
+import java.util.Map;
+import javax.crypto.SecretKey;
+import javax.crypto.spec.SecretKeySpec;
+import lombok.experimental.UtilityClass;
+
+/**
+ * Utility class containing JWE decryption related methods.
+ */
+@UtilityClass
+public class JWEDecrypterUtil {
+
+    public byte[] decrypt(
```

Comment:
Please add doc-comment for this method and also specify the exceptions which the method can throw.

Reply:
Added